src/components/common/container.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react'
-import '../../scss/components/common/container.scss'
+import containerStyles from '../../scss/components/common/container.module.scss'
```

Comment:
When importing style modules, is there a benefit to assigning a component-specific name to the import? That is, using `containerStyles` or `detailTextStyles` instead of something generic like `styles`?

(I haven’t gatsby’d before; I’m just asking for my edification here!)

Reply:
I think I just always err on long variable names, plus some pages *could* have multiple sheets in the future.